utils/docker/Makefile

```diff
@@ -23,7 +23,8 @@ DOCKER_DEV_BUILD_OPTS ?=
 ifdef DOCKER_DEV_IMAGE_APT_ADD__PKGS
 	DOCKER_DEV_BUILD_OPTS += --build-arg APT_ADD_PKGS=$(DOCKER_DEV_IMAGE_APT_ADD__PKGS)
 endif
-
+# needed because we're doing the apply of fabric patches inside of docker, where git is unconfigured
+DOCKER_DEV_BUILD_OPTS += --build-arg GIT_USER_NAME=$(shell git config user.name) --build-arg GIT_USER_EMAIL=$(shell git config user.email)
```

Comment:
These need to be quoted. My name is "Morgan Bauer" and the space breaks this.

--build-arg GIT_USER_NAME="$(shell git config user.name)" --build-arg GIT_USER_EMAIL="$(shell git config user.email)"